### ISSUE TITLE
Kernel fails to download / install

### ISSUE THREAD

**Issue by zsoltkraft** [bkw777/mainline#217]
Downloading / installing kernels started to fail. It used to work without issues. Tried reinstalling the mainline package, not much changed. Installing older kernels fails the same way.

```
$ sudo mainline --install-latest
mainline 1.1.9
Distribution: Linux Mint 21.1
Architecture: amd64
Running kernel: 6.2.8-060208-generic
Updating from: 'https://kernel.ubuntu.com/~kernel-ppa/mainline/'
OK
Fetching individual kernel indexes...
Found installed : 6.2.7-060207.202303170542                                    
Found installed : 6.2.8-060208.202303220943
Latest point update: 6.2.9

Install Kernel Version 6.2.9 ? (y/n): 
y

** (process:12939): CRITICAL **: 15:48:56.077: download_item_construct: assertion 'cksum != NULL' failed

** (process:12939): CRITICAL **: 15:48:56.077: download_task_add_to_queue: assertion 'item != NULL' failed

** (process:12939): CRITICAL **: 15:48:56.077: download_item_construct: assertion 'cksum != NULL' failed

** (process:12939): CRITICAL **: 15:48:56.077: download_task_add_to_queue: assertion 'item != NULL' failed

** (process:12939): CRITICAL **: 15:48:56.077: download_item_construct: assertion 'cksum != NULL' failed

** (process:12939): CRITICAL **: 15:48:56.077: download_task_add_to_queue: assertion 'item != NULL' failed

** (process:12939): CRITICAL **: 15:48:56.077: download_item_construct: assertion 'cksum != NULL' failed

** (process:12939): CRITICAL **: 15:48:56.077: download_task_add_to_queue: assertion 'item != NULL' failed
Downloading 6.2.9
```

**Comment by Danik2343** [comment]
Yes this glitch

**Comment by bkw777** [comment]
Fixed already in 1.1.10
If you're waiting for the ppa packages to update, then in the meantime the work-around is go into settings and enable Verify Checksums.

It's a new feature which is disabled by default because it's new, but had a bug when disabled.

The feature downloads the CHECKSUMS file and verifies the sha256 checksums when downloading the .deb files.

**Comment by bkw777** [comment]
ppa has 1.1.11 now
